install/bootstrap.sh

```diff
@@ -29,28 +59,18 @@ if [[ $OSTYPE == "linux-gnu" && $CLOUD_SHELL == true ]]; then
     echo "🛠 Installing client tools."
     ./common/install-tools.sh
 
-    echo "🚪 Configuring Cloud Shell to re-init environment if disconnected."
+    if [ $(grep "bank-of-anthos-init" ~/.bashrc | wc -l) -eq 0 ]
+    then
+      echo "🚪 Configuring Cloud Shell to re-init environment if disconnected."
+      echo "source $ROOT/bank-of-anthos-scripts/install/bank-of-anthos-init.sh" >> ~/.bashrc
+    fi
     echo "source $ROOT/bank-of-anthos-scripts/install/env" >> ~/.bashrc
     echo "source $ROOT/bank-of-anthos-scripts/install/common/install-tools.sh" >> ~/.bashrc
 
 
     echo "🔆 Enabling GCP APIs. This may take up to 5 minutes."
-    gcloud services enable \
-    container.googleapis.com \
-    compute.googleapis.com \
-    stackdriver.googleapis.com \
-    meshca.googleapis.com \
-    meshtelemetry.googleapis.com \
-    meshconfig.googleapis.com \
-    iamcredentials.googleapis.com \
-    anthos.googleapis.com \
-    cloudresourcemanager.googleapis.com \
-    gkeconnect.googleapis.com \
-    gkehub.googleapis.com \
-    serviceusage.googleapis.com \
-    sourcerepo.googleapis.com \
-    cloudbuild.googleapis.com \
-    secretmanager.googleapis.com
+    local APIS="container compute stackdriver meshca meshtelemetry meshconfig iamcredentials anthos cloudresourcemanager gkeconnect gkehub serviceusage sourcerepo cloudbuild secretmanager"
```

Comment:
Should 'local' be here as well as the function - thought that should be valid within the function branch itself? 
Just need to set the variable "APIS"

Reply:
No harm I guess Charlie - just trying to restrict the scope of the APIS variable to within the bootstrap script and not persist after the script is finished. Making it local makes it available to the sub function anyway, so in reality I probably don't need to pass it in as an arg if we're being strict I guess.